src/pages/profile/edit/personal-info/index.tsx

```diff
@@ -0,0 +1,121 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import EditablePanel from '../EditablePanel';
+import FullName from './EditFullName';
+import Gender from './EditGender';
+import Country from './EditCountry';
+import Description from './EditDescription';
+import PhoneNumber from './EditPhoneNumber';
+import { useAppSelector } from '../../../../hooks/redux-hooks';
+import AddImage from '../../AddImage';
+import { useEffect, useState } from 'react';
+import useEditAccountMutation from '../../../../api/mutations/account/useEditAccountMutation';
+import LoadingPrimary from '../../../../components/LoadingPrimary';
+
+function PersonalInfo() {
+  const user = useAppSelector((state) => state.auth.user);
+  const profile = useAppSelector((state) => state.auth.user?.Profile);
+  const profileId = profile?.id ?? '';
+
+  const { mutate } = useEditAccountMutation(profileId);
+  const [newProfileImage, setNewProfileImage] = useState('');
+
+  const fullname = user && user.firstName + ' ' + user.lastName;
```

Comment:
to not check user and profile twice you can do like this:
if (!user || !profile) return <LoadingPrimary 

const {firstname, lastname} = user
const fullname = ....
const profileId = profile.id;

return  <>
      <Typography
...